Makefile

```diff
@@ -42,3 +41,22 @@ test-gator-dockerized: __build-gator
 .PHONY: build-gator
 __build-gator:
 	docker build --build-arg GATOR_VERSION=$(GATOR_VERSION) -f build/gator/Dockerfile -t gator-container .
+
+.PHONY: yq
+yq:
+	# Download and install yq
+	sudo snap install yq
```

Comment:
Is snap available to people running this locally?

Also, it looks like yq may no longer be used?

Reply:
I was simply moving this line from `make integration-bootstrap`. It also looks like the entire Makefile assumes that we are running in the GitHub Actions environment (e.g. the use of `GITHUB_WORKSPACE`). I can open a follow-up PR to make it more developer-friendly.

Reply:
Yes, yq is no longer being used. Let me remove that.